_includes/2020/templates/Head.php

```diff
@@ -4,8 +4,14 @@
   namespace Keyman\Site\com\keyman\templates;
 
 use Keyman\Site\com\keyman\Util;
+use Keyman\Site\com\keyman\KeymanComSentry;
 use Keyman\Site\Common\KeymanHosts;
 
+// *Don't* use autoloader here because of potential side-effects in older pages
```

Comment:
Does this only apply to the keyman.com site?

Reply:
If it isn't there in the other sites, then no.